pkg/cli/expose/expose.go

```diff
@@ -160,7 +160,7 @@ func (o *ExposeOptions) Validate(cmd *cobra.Command) error {
 	mapping := info.ResourceMapping()
 
 	switch mapping.GroupVersionKind.GroupKind() {
-	case kapi.Kind("Service"):
+	case metav1.Kind("Service"):
```

Comment:
This is wrong, change the entire switch to:
```
switch mapping.Resource.GroupResource() {
case corev1.Resource("service"):
```
The import above needs to change accordingly.

Reply:
got it, done